inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -371,13 +373,17 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
         VPU_LOGGER_SECTION(env.log);
 
         auto convertNetwork = [&convertedNetwork, &originalOrConvertNetwork]() {
+            const auto transformations_callback = [](const std::shared_ptr<const ::ngraph::Node> &node) -> bool {
```

Comment:
Name it predicate, please.
Also, comment describing why we don't want to process Gelu could be useful here

Reply:
As you could see, I did it. But then I saw the name of the argument:
```
explicit ConvertOpSet3ToOpSet2(const PassParam::param_callback & callback = PassParam::getDefaultCallback())
            : FunctionPass(), PassParam(callback) {}
```  
Do you think this is more consistent than `callback`?  Also, it seems to me that this `callback `only asks the caller a question, and does not mean the order of calling certain functions.

> Also, comment describing why we don't want to process Gelu could be useful here  
It's a good idea. Something like? 
`We are notify NGraph that Myriad plugin supports Gelu and this layer should not be split into multiple layers. `

Reply:
`callback` - it something that is triggered after event happened, `predicate` - criteria for triggering event. So I thin `predicate` fits here. Indeed, in function it's named other way, but I see it as code style bug and don't think we should inherit it.

I thought something about:

```cpp
// disable GeLU decomposition
```

Reply:
Ok, I would only say that all plugins use `transformations_callback`:)

Reply:
@gladilov-gleb this `function` is called after callback inside transformation is called, so it's a callback based on your description.

Reply:
If it's called after transformation which way it prevents from GeLU decomposition?
Anyway, it returns true or false which defines will we run transformation or not, right? If my understand is correct here, I still see that as `predicate`